include/ord/OpenRoad.hh

```diff
@@ -247,6 +247,9 @@ class OpenRoad
   void addObserver(OpenRoadObserver* observer);
   void removeObserver(OpenRoadObserver* observer);
 
+  const char* getVersion() const;
+  const char* getGitDescribe() const;
```

Comment:
These could be static methods as they don't depend on any memberes afaik.

Reply:
makes sense, I'll switch them to static